### DISCUSSION TITLE
Mods Do not Load after update to .NET SDK

### DISCUSSION THREAD

**Discussion by JoewAlabel** [BepInEx/BepInEx#463] [Help and Q&A] [answered]
I updated my mod from .NET FRAMEWORK to .NET 6.0, compiled with no errors.
Placed the DLL inside BepInEx\plugins folder's.

But it tells me 0 mods to be loaded and 0 mods loaded -.-

What am i missing, please?
And where are the class PluginInfo, please? I am using VS 2022, pressed F2 to go to the file, edit it, but after i compile, it removes all my changes from the file.

Ty!!!!

`[Message: Preloader] BepInEx 6.0.0-be.649 - Rune Factory 5
[Message: Preloader] Built from commit 78b5b58884fa3410c7d43f151172f09c11aa3766
[Info   :   BepInEx] System platform: Windows 10 64-bit
[Info   :   BepInEx] Process bitness: 64-bit (x64)
[Info   :   BepInEx] Running under Unity 2019.4.17f1
[Info   :   BepInEx] Runtime version: 6.0.7
[Info   :   BepInEx] Runtime information: .NET 6.0.7
[Info   : Preloader] 0 patcher plugins loaded
[Info   : Preloader] 0 assemblies discovered
[Message:AssemblyPatcher] Executing 0 patch(es)
[Message:   BepInEx] Chainloader initialized
[Info   :Il2CppInterop] Registered mono type Il2CppInterop.Runtime.DelegateSupport+Il2CppToMonoDelegateReference in il2cpp domain
[Info   :   BepInEx] 0 plugins to load
[Message:   BepInEx] Chainloader startup complete`

**Comment by JoewAlabel** [comment, marked as answer]
I created a new project using the right template and worked. For some reason i created with wrong template.

**Reply by super-miner** [reply]
Can you please tell me what template you used that fixed it? I am having the same problem.

Edit: 
I figured it out, I was using the "bep6plugin_unity_il2cpp" template instead of "bep6plugin_il2cpp". I also installed the 1.4.0 version of the templates instead of the 2.0.0-be.1 version.
